index.html

```diff
@@ -7,40 +7,75 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
-      text-align: center;
+      text-align: left;
```

Comment:
Thanks for all of the feedback. Regarding design, I completely agree. I intend to put more time/energy into that element in the future. Not making excuses, but the git push was hanging me up for quite some time because I missed the first class. Turns out I was not logged in. Related, I have since received your invitation to the course organization on GitHub. 

I'll be starting earlier in the week on the assignments so we're not having back and fourth right before the next class.  You'll get a better product in the future.

Looking forward to meeting you this afternoon. - Brad